Dockerfile

```diff
@@ -1,7 +1,24 @@
-FROM golang:latest
+# builder
+FROM golang:latest AS builder
 RUN mkdir /app
 ADD . /app/
 WORKDIR /app
+
 RUN go mod download
-RUN go build -o main .
-CMD ["/app/main", "--portHTTP=80"]
\ No newline at end of file
+RUN make build
```

Comment:
Not sure how docker on windows works, but will this command run inside of a container or no the host system?
I wanna keep everything as compatible as possible.

Reply:
The command will be run inside the docker container, like when you're running an `apt get whatever`.